pkgs/development/python-modules/harlequin-mysql/default.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+  python3Packages,
+}:
+
+buildPythonPackage rec {
+  pname = "harlequin-mysql";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "harlequin_mysql";
+    inherit version;
+    hash = "sha256-Ru9CxbZYVo9TQO5TwkHLEzPz4EkUgHwfg3Qeg1F4eLM=";
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
+  dependencies = with python3Packages; [
+    mysql-connector
+  ];
+
+  pythonRelaxDeps = [
+    "mysql-connector-python"
+  ];
+
+  # To prevent circular dependency
+  # as harlequin-mysql requires harlequin which requires harlequin-mysql
+  doCheck = false;
+  pythonRemoveDeps = [
+    "harlequin"
+  ];
+
+  meta = {
+    description = "A Harlequin adapter for MySQL";
```

```suggestion
    description = "Harlequin adapter for MySQL";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description
`meta.description` should not be started with the definite or an indefinite article.